utils/serialise.js

```diff
@@ -1,12 +1,13 @@
 module.exports = serialise
 
-function serialise (type, attributes, id) {
+function serialise (type, attributes, id, createdAt) {
   if (!type || !attributes) {
     throw new Error('Serialisation must include a type and some attributes.')
   }
   var data = {
     type: type,
-    attributes: attributes
+    attributes: attributes,
+    createdAt: createdAt
   }
```

Comment:
Made the changes to set `options.createdAt = new Date().toISOString()` in `lib/sign-up.js` and everything seems fine, except one of the unit tests that's testing the response https://github.com/hoodiehq/hoodie-account-client/blob/master/test/unit/sign-up-test.js#L60-#L71